gtk/src/subclass/mod.rs

```diff
@@ -33,6 +33,11 @@ pub mod widget;
 pub mod window;
 
 pub mod prelude {
+    #[doc(hidden)]
+    pub use gio::subclass::prelude::*;
+    #[doc(hidden)]
+    pub use glib::subclass::prelude::*;
```

Comment:
What does it allow to do btw?

Reply:
Not having to import all the subclass preludes every time. Same as what was done for the normal preludes.

Reply:
I never used subclass so I really can't judge here. I'll trust you here. :)